src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -472,7 +472,7 @@
             </Grid.Resources>
             <graphControl:Grapher Name="GraphingControl"
                                   Style="{ThemeResource ThemedGrapherStyle}"
-                                  ForceProportionalAxes="True"
+                                  ForceProportionalAxes="False"
```

Comment:
Do we want to remove the logic In GraphingSettingsViewModel.cpp, UpdateDisplayRange() where ForceProportionalAxes is used?

Reply:
removed!